bootstrap4-subpage.html

```diff
@@ -41,7 +41,7 @@ <h4 class="project-headline text-center">tech stack</h4>
 
                         </ul>
                         <a href="https://github.com/DamianJorga/learning-bootstrap4" class="project--link text-center text-uppercase" target="_blank">project code</a>
-                        <a href="https://damianjorga.github.io/learning-bootstrap4/" class="project--link text-center text-uppercase" target="_blank">demo</a>
+                        <a href="https://damianjorga.github.io/learning-bootstrap4-/" class="project--link text-center text-uppercase" target="_blank">demo</a>
```

Comment:
Do linku zewnetrznego dodaje sie zawsze atrybut rel="noopener noreferrer"

https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/